app/models/wiki_page.rb

```diff
@@ -44,14 +44,17 @@ def to_json
     }
   end
 
+  def should_create_edit?
+    WikiPageEdit.where(wiki_page: self).count == 0 || text_was != text
+  end
+
   def create_edit
-    return if text_was == text
+    return unless should_create_edit?
     raise "Cannot update WikiPage with updater not specified" if @updater.blank?
 
     diff = Diffy::Diff.new(text_was, text)
-    WikiPageEdit.create(
+    wiki_page_edits << WikiPageEdit.new(
```

Comment:
This is how we let AR figure out how to do the handoff between a parent being created and the associated record we want to create along with it